auto generated update of versioned plugin documentation with manual fix (#1325)
elastic/logstash-docs: docs/versioned-plugins/inputs/beats-v6.3.0.asciidoc

&nbsp;

[id="{version}-plugins-{type}s-{plugin}-add_hostname"]
===== `add_hostname`

  deprecated[6.0.0, The default value has been changed to `false`. In 7.0.0 this setting will be removed]

  * Value type is {logstash-ref}/configuration-file-structure.html#boolean[boolean]
  * Default value is `false`

Flag to determine whether to add `host` field to event using the value supplied by the {plugin-singular} in the `hostname` field.

[id="{version}-plugins-{type}s-{plugin}-cipher_suites"]
===== `cipher_suites`

  * Value type is {logstash-ref}/configuration-file-structure.html#array[array]
  * Default value is `java.lang.String[TLS_AES_128_GCM_SHA256, TLS_AES_256_GCM_SHA384, TLS_CHACHA20_POLY1305_SHA256, TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384, TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384, TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256, TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256, TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256, TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256, TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA384, TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384, TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256, TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256]@459cfcca`

The list of ciphers suite to use, listed by priorities.

> The default values applies for OpenJDK 11.0.14 and higher, for older versions the list does not include suites not supported by the JDK, such as the ChaCha20 family of ciphers.

Reviewer: ```suggestion
This default list applies for OpenJDK 11.0.14 and higher.
For older JDK versions, the default list includes only suites supported by that version.
For example, the ChaCha20 family of ciphers is not supported in older versions.
```

:plugin: beats
:type: input
:default_codec: plain
:plugin-uc: Beats
:plugin-singular: Beat

///////////////////////////////////////////
START - GENERATED VARIABLES, DO NOT EDIT!
///////////////////////////////////////////
:version: v6.3.0

> :release_date: 2022-03-21

Reviewer: Note to self (@karenzone):  Beats changes necessitates change to input-elastic_agent to bring it into alignment.
